src/main/java/net/fabricmc/mappingio/tree/MappingTreeView.java

```diff
@@ -24,6 +24,9 @@
 
 import net.fabricmc.mappingio.MappingVisitor;
 
+/**
+ * Immutable mapping tree.
```

Comment:
```suggestion
 * Read-only mapping tree.
```
The main implementation is definitely not immutable.